libs/ts-rest/core/src/lib/dsl.ts

```diff
@@ -23,6 +23,7 @@ export type AppRouteQuery = {
   description?: string;
   deprecated?: boolean;
   responses: Record<number, unknown>;
+  strict?: boolean;
```

Comment:
Can this be a more descriptive name, maybe `strictResponses`?

Reply:
Actually that's slightly ambiguous too because it might be mistaken to do the same functionality of `validateResponses`. Maybe `strictStatusCodes`?

Reply:
Changed it to `strictStatusCodes`